code/game/objects/items/climbingrope.dm

```diff
@@ -0,0 +1,86 @@
+/obj/item/climbing_hook
+	name = "climbing hook"
+	desc = "Standard hook with rope to scale up holes. The rope is of average quality, but due to your weight amongst other factors, may not withstand extreme use."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "climbingrope"
+	inhand_icon_state = "crowbar_brass"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	force = 5
+	throwforce = 10
+	reach = 2
+	throw_range = 4
+	w_class = WEIGHT_CLASS_NORMAL
+	attack_verb_continuous = list("whacks", "flails", "bludgeons", "swings")
```

Comment:
Wouldn't "swings" here read as `"John Mellons swings Timothy Validson with the climbing hook"`?
That doesn't sound right